improver/regrid/landsea.py

```diff
@@ -223,6 +225,12 @@ def process(
         )
 
 
+class NoMatchesError(ValueError):
```

Comment:
👍

I'm not sure on the standard approach for custom error messages in the long run, should we put them in a central location? Here's fine for this one for now I think

Reply:
I agree that a cetralised location for custom public exceptions is a good move.

However this particular exception would not be included as one of them - it's only raised by this private `AdjustLandSeaPoints._get_matches` method, caught by the public method `correct_where_input_true` which calls it.  That is, it is an internal implementation detail that isn't exposed to the end user.

Tell you what, I'll add a leading underscore to the name to denote that the custom exception is 'private' and also move it in as a class attribute.